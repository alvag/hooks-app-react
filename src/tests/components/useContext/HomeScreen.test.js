import React from 'react';
import { shallow, mount } from 'enzyme';
import HomeScreen from '../../../components/useContext/HomeScreen';
import { UserContext } from '../../../components/useContext/UserContext';

describe( 'Pruebas en <HomeScreen />', () => {

    const user = {
        name: 'Max',
        email: 'max@gmail.com'
    };

    const wrapper = mount(
        <UserContext.Provider value={{
            user
        }}>
            <HomeScreen/>
        </UserContext.Provider>
    );

    test( 'debe mostrar el componente correctamente', () => {
        expect( wrapper ).toMatchSnapshot();
    } );


} );