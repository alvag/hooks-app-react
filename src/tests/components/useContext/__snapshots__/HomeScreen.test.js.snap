// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen /> debe mostrar el componente correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {"name":"Max","email":"max@gmail.com"}
  </pre>
</div>
`;
